src/Helper/Helper_PDF_List_Table.php

```diff
@@ -353,7 +353,7 @@ public function column_template( $item ) {
 	 * @since 4.0
 	 */
 	public function column_name( $item ) {
-		$edit_url        = add_query_arg( [ 'pid' => $item['id'] ] );
+		$edit_url        = esc_url( add_query_arg( [ 'pid' => $item['id'] ] ) );
```

Comment:
@jestonihpi this would be better handled at the point the URL is output in the HTML.